apps/mobile/src/features/send/send-form/components/send-form-amount-field.tsx

```diff
@@ -6,6 +6,7 @@ import { z } from 'zod';
 import { SendFormBaseContext, useSendFormContext } from '../send-form-context';
 
 export function SendFormAmountField<T extends SendFormBaseContext<T>>() {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why do we need to disable lint for `formData`? 

I see `eslint` gives this error: `'formData' is assigned a value but only used as a type` . Maybe we can fix that instead?

Reply:
I assume this is work in progress by @fbwoolf, definitely worth tightening it up, but probably out of scope for this PR.